components/UI/connectButton.tsx

```diff
@@ -20,7 +20,9 @@ const ConnectButton: FunctionComponent = () => {
   }, [isConnected]);
 
   const connectWallet = async (connector: Connector) => {
-    await connectAsync({ connector });
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    await connectAsync({ connector: connector });
```

Comment:
really needed ?

Reply:
Yes it doesn't build because of a linter error

Reply:
I think we updated our configuration/linter on Vercel, because these errors where already there, and should have in theory blocked any PR from being deployed

Reply:
The latest versions of starknet react and starknetkit have incompatible types, but work well (as it used to do) in practice. We can either wait for the next update, or I can also ask Cecco if there is anything to fix this type issue.